content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>We could use a link on "reverse-mode differentiation" if you have one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Good idea @melissawm. 

This is the closest reference I could find on Wikipedia:

```diff
... and backpropagation (
- reverse-mode
+ [reverse-mode](https://en.wikipedia.org/wiki/Automatic_differentiation#Reverse_accumulation)
differentiation)
...
```